<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_question</class>
 <widget class="QDialog" name="d_question">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#d_question {
    border-image: url(:/bg/resource/bg_most_2.png);
}

#pushButton_question_return{
    border-image: url(:/icons/resource/back.png);
}

#pushButton_question_return:hover{
    border-image: url(:/icons/resource/back_h.png);
}
#label_question_text1,#label_question_Quser{
    color: white;
    font-family: &quot;黑体&quot;;
}

#label_question_title,#label_question_Time{
    color: #F5EEEE;
}

#textBrowser_question_text{
    border: none;
    background-color: rgb(146,168,209);
    color: white;
}

#pushButton_question_Prev{
    border-image: url(:/icons/resource/Arrow_Left_Default.png);
}

#pushButton_question_Prev:hover{
    border-image: url(:/icons/resource/Arrow_Left_Hover.png);
}

#pushButton__question_Next{
    border-image: url(:/icons/resource/Arrow_Right_Default.png);
}

#pushButton__question_Next:hover{
    border-image: url(:/icons/resource/Arrow_Right_Hover.png);
}

#pushButton__question_Write{
    border-image: url(:/icons/resource/pen.png);
}

#pushButton__question_Write:hover{
    border-image: url(:/icons/resource/pen_h.png);
}

#listWidget_question_ansList{
	border: none;
    background: #F5F1F1
}</string>
  </property>
  <widget class="QListWidget" name="listWidget_question_ansList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>451</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_question_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>437</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_question_Quser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>14</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_question_focus">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>3</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_question_Prev">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton__question_Write">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>660</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton__question_Next">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>660</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_question_return">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_question_text1">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>3</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>提问者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_question_Time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_question_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
